src/Moryx/Serialization/EntryConvert/EntryConvert.cs

```diff
@@ -495,6 +495,26 @@ private static Entry ConvertParameter(ParameterInfo parameter, ICustomSerializat
             return parameterModel;
         }
 
+        /// <summary>
+        /// Convert string[] possible values to EntryPossible[].
+        /// Key and DisplayName are set to the string value.
+        /// </summary>
+        private static EntryPossible[] ConvertPossible(string[] possible)
```

Comment:
I think we should move the convertion one level up already, i.e. the API of the `serialization.PossibleValues(parameterType, parameter)` should already return the `EntryPossible`, don't you think? 🤔

Reply:
I also think so, `ICustomSerialization` should return `EntryPossible`